api.js

```diff
@@ -9,6 +9,7 @@ export const getTopics = () => {
 };
 
 export const getArticles = (topic) => {
+
  return myApi.get(`/articles?${topic}`).then(({ data }) => {
     return data;
```

Comment:
You get potentially use axios params here. It works as you've got it though, so will leave it up to you if you want to refactor 👍